### PR TITLE
Update tables

### PR DESCRIPTION
**Pull request by limavfabio**
Add a column species of type string to your animals table. Modify your schema.sql file.

Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.:

set species to unspecified and rollback

Inside a transaction:
Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
Commit the transaction.
Verify that change was made and persists after commit.
Take a screenshot of the results of your actions.:

Set species

Now, take a deep breath and... Inside a transaction delete all records in the animals table, then roll back the transaction.
After the rollback verify if all records in the animals table still exists. After that, you can start breathing as usual ;)
Take a screenshot of the results of your actions.

delete and rollback

Inside a transaction:
Delete all animals born after Jan 1st, 2022.
Create a savepoint for the transaction.
Update all animals' weight to be their weight multiplied by -1.
Rollback to the savepoint
Update all animals' weights that are negative to be their weight multiplied by -1.
Commit transaction
Take a screenshot of the results of your actions:

Change negative weights to positive

Write queries to answer the following questions:
How many animals are there?
How many animals have never tried to escape?
What is the average weight of animals?
Who escapes the most, neutered or not neutered animals?
What is the minimum and maximum weight of each type of animal?
What is the average number of escape attempts per animal type of those born between 1990 and 2000?
Take a screenshot of the results of your queries:

Queries

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -8,3 +8,119 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
...
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/* Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction. */
+BEGIN;
+
+UPDATE animals
+	SET species = 'unspecified';
+
+ROLLBACK;
+
```
Comment (cynthiainga) [inline queries.sql:19]: - [ ] After updating the animals table you should select species from animals to verify if the change was made 😉 
- [ ] Also after rollback you should select species from animals to verify if the change was made 🙃

File: queries.sql [hunk 2]
```diff
@@ -8,3 +8,119 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
...
+UPDATE animals
+	SET species = 'pokemon'
+WHERE species IS NULL;
+
+SELECT * FROM animals;
+
+/*  Commit the transaction. */
+COMMIT;
+
```
Comment (cynthiainga) [inline queries.sql:39]: - [ ] It's better to select species from animals after committing to verify if the change was made 🙃 🙃

File: queries.sql [hunk 3]
```diff
@@ -8,3 +8,119 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
...
+WHERE escape_attempts = 0;
+
+/* What is the average weight of animals */
+
+SELECT AVG(weight_kg) AS average_weight_kg
+FROM animals;
+
+/* Who escapes the most, neutered or not neutered animals? */
+
+SELECT MAX(escape_attempts) FROM animals GROUP BY neutered;
```
Comment (cynthiainga) [inline queries.sql:111]: - [ ] I think you miss something in your query, you should select `neutered` 😁 😁

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @limavfabio, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] Descriptive PR ✔️ 
- [ ] Good commit history ✔️ 
- [ ] Correct Gitflow ✔️ 

## Required Changes ♻️

- [ ] Please kindly customize your README file and add the necessary information inside 🙃 🙃 
- [ ] Please kindly check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by descholar-ceo — APPROVED** [review]
Hi @limavfabio 🤝 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 Implemented all of the requirements ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**